data/mods/Aftershock/maps/terrain_groundxeno.json

```diff
@@ -415,5 +412,18 @@
     "required_str": 32,
     "flags": [ "NOITEM", "MINEABLE", "BASHABLE", "BLOCK_WIND" ],
     "bash": { "str_min": 64, "str_max": 160, "sound": "smash!", "sound_fail": "thump." }
+  },
+  {
+    "type": "terrain",
+    "id": "t_moxfloor",
+    "name": "bleached coral floor",
+    "description": "Small outgrowths of white coral spring from a underlying lattice of black pulsating veins.",
```

Comment:
"black pulsating" -> "black, pulsating" (unless they have an aura of blackness that's pulsing, or something like that)